examples/mongodb-atlas/mongodb-atlas-blueprint.yaml

```diff
@@ -14,6 +14,35 @@ actions:
       args:
         namespace: "{{ .Object.metadata.namespace }}"
         image: <registry>/<repository>/mongodb-atlas:<tag_name>
+        podOverride:
+          containers:
+          - name: container
+            env:
+            - name: MONGODB_ATLAS_PUBLIC_API_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: mongo-atlas-secret
+                  key: publickey
+            - name: MONGODB_ATLAS_PRIVATE_API_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: mongo-atlas-secret
+                  key: privatekey
```

Comment:
We should mount only certs with podOverride. Other fields can be saved and passed as output artifacts

Reply:
added all fields as output